formats/config/fixtures_test.go

```diff
@@ -0,0 +1,92 @@
+package config_test
+
+import "gopkg.in/src-d/go-git.v4/formats/config"
+
+type Fixture struct {
+	Text   string
+	Raw    string
+	Config *config.Config
+}
+
+var fixtures = []*Fixture{
+	{
+		Raw:    "",
+		Text:   "",
+		Config: config.New(),
+	},
+	{
+		Raw:    ";Comments only",
+		Text:   "",
+		Config: config.New(),
+	},
+	{
+		Raw:    "#Comments only",
+		Text:   "",
+		Config: config.New(),
+	},
+	{
+		Raw:    "[core]\nrepositoryformatversion=0",
+		Text:   "[core]\n\trepositoryformatversion = 0\n",
+		Config: config.New().AddOption("core", "", "repositoryformatversion", "0"),
+	},
+	{
+		Raw:    "[core]\n\trepositoryformatversion = 0\n",
+		Text:   "[core]\n\trepositoryformatversion = 0\n",
+		Config: config.New().AddOption("core", "", "repositoryformatversion", "0"),
+	},
+	{
+		Raw:    ";Commment\n[core]\n;Comment\nrepositoryformatversion = 0\n",
+		Text:   "[core]\n\trepositoryformatversion = 0\n",
+		Config: config.New().AddOption("core", "", "repositoryformatversion", "0"),
+	},
+	{
+		Raw:    "#Commment\n#Comment\n[core]\n#Comment\nrepositoryformatversion = 0\n",
+		Text:   "[core]\n\trepositoryformatversion = 0\n",
+		Config: config.New().AddOption("core", "", "repositoryformatversion", "0"),
+	},
+	{
+		Raw: `
+			[sect1]
+			opt1 = value1
+			[sect1 "subsect1"]
+			opt2 = value2
+		`,
+		Text: `[sect1]
+	opt1 = value1
+[sect1 "subsect1"]
```

Comment:
Yeah, tabs at the left are preserved when using `, and I need no tabs at the left.